### ISSUE TITLE
Failed to execute 'getComputedStyle' on 'Window' in QTabs

### ISSUE THREAD

**Issue by Sumurai8** [quasarframework/quasar#1063]
Quasar: 0.14.6
Vue: 2.5.2
OS: Windows 10
Node: 6.11.4
NPM: 6.11.4
Browsers: Chrome 61.0.3163.100
Any other software related to your bug:

### What did you get as the error?
TypeError: Failed to execute 'getComputedStyle' on 'Window' in the __redraw function from QTabs

### What were you expecting?
The problem is that the __redraw function on QTabs seems to be delayed until after the route change, executing twice on the second component. The first execution of the function tries to get the width of `this.$refs.scroller`, which is now undefined. This causes the error `Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.`. Quasar should cancel the redraw, or cut the redraw short when the scroller is undefined.

### What steps did you take, to get the error?
Have a standard quasar pwa with two components:

```
<template>
  <q-tabs slot="footer">
    <q-tab
      slot="title"
      name="continue"
      exact
    >
      Continue to Dashboard
    </q-tab>
  </q-tabs>
</template>

<script>
import {
  QTabs,
  QTab
} from 'quasar'

export default {
  name: 'comp-a',

  components: {
    QTabs,
    QTab
  },

  mounted () {
    this.$nextTick(() => {
      this.$router.push('/b')
    })
  },
}
</script>
```

and

```
<template>
  <q-tabs slot="footer">
    <q-tab>
      Something
    </q-tab>
    <q-tab>
      Something
    </q-tab>
    <q-tab>
      Something
    </q-tab>
    <q-tab>
      Something
    </q-tab>
  </q-tabs>
</template>

<script>
import {
  QTabs,
  QTab
} from 'quasar'

export default {
  name: 'comp-b',

  components: {
    QTabs,
    QTab
  }
}
</script>
```

And have a router with the following routes:
```
  routes: [
    { path: '/', component: A },
    { path: '/b', component: B }
  ]
```

**Comment by dexif** [comment]
On 0.14.3 too.

**Comment by rstoenescu** [comment]
Hi,

Thanks for the detailed description! Would you be so kind to do a PR on this if you have time? This would speed up things. I'm currently 200% overloaded. Will get to this in a few days if nobody has time for it. Thanks.

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.7.
